tests/console/docker.pm

```diff
@@ -100,8 +100,12 @@ sub run {
     die("error: missing container $container_name") unless ($output_containers =~ m/$container_name/);
 
     # containers state can be saved to a docker image
-    my $output = script_output("docker container exec $container_name zypper -n in curl", 300);
-    die('error: curl not installed in the container') unless ($output =~ m/Installing: curl.*done/);
+    my $output = script_run("docker container exec $container_name zypper -n in curl", 300);
```

Comment:
To avoid confusion here, please change the var name here to $exit_code

Reply:
new commit updated to use exit_code var name as suggested